homebrew_challenge/src/components/SingleValue/SingleValueBar.test.js

```diff
@@ -2,7 +2,7 @@
 
 import React from "react";
 import SingleValueBar, {
-  parseCsvData,
+  // parseCsvData,
```

Comment:
Dead code - should be deleted

Reply:
Done! Will be pushed today with my changes I mentioned in a comment above.